### DISCUSSION TITLE
Regarding socket connection using unix

### DISCUSSION THREAD

**Discussion by arjunvk1027** [BretFisher/ama#212] [Q&A]
I need to make a socket connection from my container to outside  of the container. I am doing a project involving hardware parts as well . So I need to send the outputs generated inside the container to the board or desktop. I am using UNIX . I couldn't establish connection between them . Kindly , provide me a suitable suggestion/support for this problem.

**Comment by BretFisher** [comment]
Hey there, what has your research shown? If it's file sockets have you just tried mounting the path? If it's network sockets, they should work out-of-the-box.  If you found something that works, feel free to add details here to help others.
